Sources/ViewInspector/Inspector.swift

```diff
@@ -110,10 +110,15 @@ internal extension Inspector {
 private extension String {
     func sanitizingNamespace() -> String {
         var str = self
-        while let range = str.range(of: ".(unknown context at ") {
-            let end = str.index(range.upperBound, offsetBy: .init(11))
-            str.replaceSubrange(range.lowerBound..<end, with: "")
-        }
+
+        let pattern = "\\.\\(unknown context at ..........\\)"
```

Comment:
@rafael-assis I assume that we use 10 `.`s in the regex (vs the `11` offset we see in the previous code) since we know the last character is always a `)`?

Reply:
@bachand that is correct! 👍